libcontainer/cgroups/fscommon/fscommon_test.go

```diff
@@ -9,18 +9,12 @@ import (
 	"strconv"
 	"testing"
 	"time"
-
-	"github.com/opencontainers/runc/libcontainer/cgroups"
 )
 
 func TestWriteCgroupFileHandlesInterrupt(t *testing.T) {
-	if cgroups.IsCgroup2UnifiedMode() {
```

Comment:
Shouldn't we keep this check?

`/sys/fs/cgroup/memory` can exist on v2 for a group named "memory" , so just checking `Stat("/sys/fs/cgroup/memory")` is not enough.

Reply:
This is done to remove the `cgroups/fscommon dependency on `cgroups`, as otherwise we'll have a dep cycle (introduced by the following commits in this PR).

OK, let me add a check...

Reply:
added (and commit message is expanded to explain why we do it)

Reply:
@AkihiroSuda PTAL